src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
+\s+          #
+value=       #
+['\"]        # Single or double quote.
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
+)            # End of 'import' variable.
+['\"]        # Single or double quote.
+""", re.VERBOSE)
+
+FTI_SCHEMA_PATTERN = re.compile(r"""
+\s           # Whitespace.
+name=        #
+['\"]        # Single or double quote.
+schema       #
+['\"]        # Single or double quote.
+>            # End of opening tag
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
```

Comment:
My assumption here is that the package is in a sane state, if the XML is broken, maybe is actually even a good idea to suggest broken missing imports so that they can grep for those, don't you think?